cypress/e2e/API/api_EngageSphere.cy.js

```diff
@@ -0,0 +1,160 @@
+/// <reference types="cypress" />
+
+//DÚVIDAS!!!
+//to.be.at.greaterThan(9999).and.to.be.lessThan(50000)
+//Retorna que o customer Littel  tem 10.000 funcionários e também que ele possuí menos de 50.000
+
+describe('ENGAGE SPHERE API TESTING', () => {
+
+    const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}`
+
+    it('SUCCESSFULLY RETRIEVES CUSTOMERS (E.G., CHECKS FOR THE 200 STATUS CODE)', () => {
+        cy.request('GET', CUSTOMERS_API_URL + '/customers')
+            .then(({ body, status }) => {
+                expect(status).to.be.eq(200);
+                expect(body.pageInfo.totalCustomers).to.be.is.not.null;
+
+            });
+    })
+    it('RETURNS THE CORRECT STRUCTURE OF THE RESPONSE OF THE FIRST PAGE OF CUSTOMERS(I.E., CUSTOMERS AND PAGEINFO PROPERTIES)', () => {
+        cy.request('GET', CUSTOMERS_API_URL + '/customers?page=1')
```

Comment:
Já ouviu falar na funcionalidade [_template literals_](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) do JavaScript?

Nela, em vez de usar o sinal de soma (`+`), você pode fazer algo mais legível.

Veja abaixo:

```suggestion
        cy.request('GET', `${CUSTOMERS_API_URL}/customers?page=1`)
```